dan_layer/core/src/workers/hotstuff_waiter.rs

```diff
@@ -1229,7 +1229,7 @@ where
                             })?;
                             Ok(finalize)
                         }),
-                        Err(e @ HotStuffError::MissingPledges(_)) => {
```

Comment:
This is the consensus change

Reply:
I don't have enough context to understand this comment

Reply:
1) fix(consenus): vote reject and release pledges if remote pledges invalid 
2) fix: remove extraneous new output shards from most of the walletd transactions

This change relates to 1, the rest of the changes relate to 2